WalletWasabi.Packager/Program.cs

```diff
@@ -396,6 +399,103 @@ private static void Main(string[] args)
+						var newFolderRelativePath = Path.Combine(debUsrBinFolderRelativePath, newFolderName);
+						newFolderPath = Path.Combine(binDistDirectory, newFolderRelativePath);
+						Directory.Move(publishedFolder, newFolderPath);
+
+						var assetsFolder = Path.Combine(guiProjectDirectory, "Assets");
+						var assetsInfo = new DirectoryInfo(assetsFolder);
+
+						foreach (var file in assetsInfo.EnumerateFiles())
+						{
+							var number = file.Name.Split(new string[] { "WasabiLogo", ".png" }, StringSplitOptions.RemoveEmptyEntries);
+							if (number.Count() == 1 && int.TryParse(number.First(), out int size))
+							{
+								string destFolder = Path.Combine(debUsrShareIconsFolderPath, $"{size}x{size}", "apps");
+								Directory.CreateDirectory(destFolder);
+								file.CopyTo(Path.Combine(destFolder, $"{executableName}.png"));
+							}
+						}
+
+						var controlFilePath = Path.Combine(debianFolderPath, "control");
+						// License format doesn't yet work, but should work in the future, it's work in progress: https://bugs.launchpad.net/ubuntu/+source/software-center/+bug/435183
+						var controlFileContent = $"Package: {executableName}\n" +
+													$"Priority: optional\n" +
+													$"Section: utils\n" +
+													$"Maintainer: nopara73 <adam.ficsor73@gmail.com>\n" +
+													$"Version: {versionPrefix}\n" +
+													$"Homepage: http://wasabiwallet.io\n" +
+													$"Vcs-Git: git://github.com/zkSNACKs/WalletWasabi.git\n" +
+													$"Vcs-Browser: https://github.com/zkSNACKs/WalletWasabi\n" +
+													$"Architecture: amd64\n" +
+													$"License: Open Source (MIT)\n" +
+													$"Installed-Size: {installedSizeKb}\n" +
+													$"Depends:\n" +
+													$"Description: open-source, non-custodial, privacy focused Bitcoin wallet\n" +
+													$"  Built-in Tor, CoinJoin and Coin Control features.\n";
+
+						File.WriteAllText(controlFilePath, controlFileContent, Encoding.ASCII);
+
+						var desktopFilePath = Path.Combine(debUsrAppFolderPath, $"{executableName}.desktop");
+						var desktopFileContent = $"[Desktop Entry]\n" +
+													$"Type=Application\n" +
+													$"Name=Wasabi Wallet\n" +
+													$"GenericName=Bitcoin Wallet\n" +
+													$"Comment=Privacy focused Bitcoin wallet.\n" +
+													$"Icon={executableName}\n" +
+													$"Terminal=false\n" +
+													$"Exec={executableName}\n" +
+													$"Categories=Office;Finance;\n" +
+													$"Keywords=bitcoin;wallet;crypto;blockchain;wasabi;privacy;anon;awesome;qwe;asd;\n";
+
+						File.WriteAllText(desktopFilePath, desktopFileContent, Encoding.ASCII);
+
+						var wasabiStarterScriptPath = Path.Combine(debUsrBinFolderPath, $"{executableName}");
+						var wasabiStarterScriptContent = $"#!/bin/sh\n" +
+															$"{ linuxWasabiWalletFolder.TrimEnd('/')}/./{executableName}\n";
+
+						File.WriteAllText(wasabiStarterScriptPath, wasabiStarterScriptContent, Encoding.ASCII);
```

Comment:
I did the same even when I am sure this is not necessary. It was a hack to move forward faster but I think we could come back to this launcher script in a future.